shell/platform/windows/text_input_manager_win32.cc

```diff
@@ -10,10 +10,44 @@
 
 #include <imm.h>
 
+#include <cassert>
 #include <memory>
 
 namespace flutter {
 
+// RAII wrapper for the Win32 Input Method Manager context.
+class ImmContext {
+ public:
+  ImmContext(HWND window_handle)
+      : context_(::ImmGetContext(window_handle)),
+        window_handle_(window_handle) {
+    assert(window_handle);
+  }
+
+  ~ImmContext() {
+    if (window_handle_ != nullptr && context_ != nullptr) {
```

Comment:
Why are you checking `window_handle_` when it's asserted to be non-null at construction?

Reply:
Good question! Added the assert at the last minute and forgot to remove the check from the dtor. Fixed.